specification/network/resource-manager/Microsoft.Network/stable/2020-08-01/networkManager.json

```diff
@@ -0,0 +1,710 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "NetworkManager",
+    "description": "The Microsoft Azure Virtual Network Manager API provides a RESTful set of web services that interact with Microsoft Azure Network Manager service to manage your network resources.",
+    "version": "2020-08-01"
```

Comment:
As you are planning to go public preview with these new resources & APIs, please use a preview api version. The current files are under a stable version which indicates the API is GA.